scandium-core/src/main/java/org/eclipse/californium/scandium/config/DtlsConnectorConfig.java

```diff
@@ -104,6 +107,8 @@
 
 	/** the supported cipher suites in order of preference */
 	private CipherSuite[] supportedCipherSuites;
+	
+	private Set<RawPublicKeyIdentity> trustedRPKs;
```

Comment:
Using a `Set<RawPublicKeyIdentity>` seems to be much less flexible than the interface `PskStore`, which may be either implemented with a simple "InMemory" implementation or something else (as long , as it is fast enough).

Reply:
+1, I also think adding an interface will be better. Maybe with a default implementation which trust anyone. (isTrusted return always true)

Reply:
Ok. I'm thinking of placing this in the auth package, or do you think it should go somewhere else?